src/server/routes/editor.tsx

```diff
@@ -73,7 +79,7 @@ router.get('/edit', auth.isAuthenticated, async (req, res, next) => {
 	// Parallel fetch the three required models. Only "editorModelPromise" has
 	// "require: true", so only that can throw a NotFoundError, which is why the
 	// other two model fetch operations have no error handling.
-	const [editorModel, titleUnlockModel, genderModel] = await Promise.all([
+	const [editorModel, titleUnlockModel, genderModel]: any = await Promise.all([
```

Comment:
The types for these models should probably be declared in `bookbrainz-data`, which is something I will probably continue to work on for my GSoC project soon. If you want, you can also have a look yourself and help to add type definitions for a few more models here: https://github.com/metabrainz/bookbrainz-data-js/tree/master/src/types

Reply:
I agree that it would be better if we can define them in `bookbrainz-data`. I'm marking this as a stretch-goal for now.